src/index.ts

```diff
@@ -34,7 +34,11 @@ export default {
   CENTER: constantsSource.CENTER,
 
   show(message: string, durationSeconds: number, styles: StylesIOS = {}) {
-    RCTToast.show(message, durationSeconds, processColors(styles));
+    RCTToast.show(
+      message,
+      durationSeconds === undefined ? constantsSource.SHORT : durationSeconds,
```

Comment:
Can you please use the same way as showWithGravity, i.e. "??"

Thank you

Reply:
sure

Reply:
fixed